oshc/main/static/main/css/app.css

```diff
@@ -160,6 +160,9 @@ iframe {
   border: 0;
 }
 
+.contest-submission-message {
+  margin-top: 15%;
```

Comment:
This doesn't work well when I decrease the size of the screen, at this moment I don't have better suggestions, just trying out things with CSS. Push if you see a better way of having this on breadth small, length long screen size, otherwise I'll do some edits and merge. 

Thanks for the work!